src/interfaces.ts

```diff
@@ -1,14 +1,9 @@
+import { InstallState } from '@aryanshridhar/fiddle-core';
```

Comment:
One merge-blocker - testing with a fork of `fiddle-core` is perfectly fine, but we need to cut a new release and use the real `fiddle-core` package before we merge this into Fiddle's main branch.

Reply:
Yup right! The main branch of `electron/fiddle-core` has all the right set of APIs required. We just need to release it :)